week-2/week-2-Assignment-2/Pizzashop.java

```diff
@@ -0,0 +1,97 @@
+public class Pizzashop {
+    private int crustthinCrustCost = 50;
+    private int crustCheeseBurstCost = 100;
+    private int crustClassicPan = 50;
+    private int toppingCornCost = 50;
+    private int toppingChickenCost = 150;
```

Comment:
Too many variables. Consider using a function to calculate the cost of each element using a conditional statement.